com.vrcfury.vrcfury/Editor/VF/Feature/ToggleBuilder.cs

```diff
@@ -180,8 +202,17 @@ VFABool physBoneResetter
 
         VFState inState;
         VFState onState;
+        VFState outState;
+
         if (model.hasTransition) {
             var transitionClipIn = actionClipService.LoadState(onName + " In", inAction);
+
+            // if clip is empty, copy last frame of transition
+            if (clip == fx.GetEmptyClip()) {

```

Comment:
This breaks restingClip, which is currently saved above this, but needs to be moved below this.